### DISCUSSION TITLE
Local layers on Qfield project are editable but the added features aren't available to push to the cloud.

### DISCUSSION THREAD

**Discussion by kcfol** [opengisch/QField#3856] [Q&A]
Hi, 

In the last month I've noticed that I am not able to "push changes" from two of the five layers in a geopackage, to the cloud, after making edits from my offline device. 

I've tested the geopackage now with the "prefer online layers" and "directly access the datasource" configuration as well as the "prefer offline layers" and "offline editing" configuration. All five layers have the same configuration and all can be edited in the qfield cloud workspace on the tablet but only three of the layers will add the features and register those features to be pushed to the cloud. 

I've also attempted to export the data.gpkg to see if the added features appear in the relevant attribute table and they aren't there either despite the "changes saved" pop up appearing. 

The same geopackage has been tested in a project with collaborators and one without collaborators. 

There aren't any errors in the message log that I can see. 

Not too sure what to do...

Thanks in advance!

**Comment by suricactus** [comment]
Hello, @kcfol . Can you give some information about the username and project name on QFieldCloud, as well as layers that are experiencing the problems?

> I've also attempted to export the data.gpkg to see if the added features appear in the relevant attribute table and they aren't there either despite the "changes saved" pop up appearing.

Can you export and attach a filename called `deltafile.json`? It is the one that records all you changes.

Finally, can you please confirm you do not experience this with another project? Try any random QGIS project you have on QFieldCloud. Also try with the same problematic project files uploaded as a new project on the cloud.

**Reply by kcfol** [reply]
Hi @suricactus, 

Overnight, I've received the QField Cloud issue report 25.01.2023 and after following the suggested instructions and re-synchronising the geopackage this morning the issue seems to have resolved. 

I didn't realise that the deltafile.json logged all of my changes so, thanks for asking for it - it's helpful to know. 

Thanks for your reply.

**Reply by suricactus** [reply]
Thanks! Glad to hear your issue is resolved!
